cmd/nfsplugin/main.go

```diff
@@ -55,6 +56,8 @@ func main() {
 	cmd.PersistentFlags().StringVar(&endpoint, "endpoint", "", "CSI endpoint")
 	cmd.MarkPersistentFlagRequired("endpoint")
 
+	cmd.PersistentFlags().StringVar(&perm, "mount-permissions", "", "mounted folder permissions")
```

Comment:
There should be either a default value or there should be no chmod if the value is empty.

Reply:
Yes, if the value is empty, we don't perform chmod: https://github.com/kubernetes-csi/csi-driver-nfs/pull/36/files#diff-e0808273996894677ae8cffcc061ed60R77